src/.nuget/Microsoft.NETCore.ILAsm/runtime.Android.Microsoft.NETCore.ILAsm.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thinking a bit more about this, why do you need to add these android specific files? Isn't the Linux version sufficient?